### PR TITLE
Added badges and fixed punctuation

### PR DESCRIPTION
**Pull request by sumitkharche**
##### ISSUE TYPE
 - Docs Pull Request

##### SUMMARY
- Added license, slack & release badges
- added period at end of sentence

Fixes
- changed in README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,12 @@
 # BotKube
+
 [![CI](https://github.com/infracloudio/botkube/workflows/CI/badge.svg?branch=develop)](https://github.com/infracloudio/botkube/actions?query=workflow%3ACI+branch%3Adevelop) [![Go Report Card](https://goreportcard.com/badge/github.com/infracloudio/botkube)](https://goreportcard.com/report/github.com/infracloudio/botkube)
 [![BotKube website](https://img.shields.io/badge/docs-botkube.io-blue.svg)](https://botkube.io)
 [![GoDoc](https://godoc.org/github.com/infracloudio/botkube?status.svg)](https://godoc.org/github.com/infracloudio/botkube)
 [![Release Version](https://img.shields.io/github/v/release/infracloudio/botkube?label=Botkube)](https://github.com/infracloudio/botkube/releases/latest)
 [![License](https://img.shields.io/github/license/infracloudio/botkube?color=light%20green&logo=github)](https://github.com/infracloudio/botkube/blob/develop/LICENSE)
+[![Slack](https://badgen.net/badge/slack/botkube/blue?icon=slack)](http://join.botkube.io/)
```
Comment (bhavin192) [inline README.md:8]:
```suggestion
[![Slack](https://badgen.net/badge/slack/BotKube/blue?icon=slack)](http://join.botkube.io/)
```

**Review by bhavin192 — CHANGES_REQUESTED** [review]
LGTM, just one minor change.

File: README.md (hunk 1 above)
Comment (sumitkharche) [reply README.md]: Done

**Review by sanketsudake — APPROVED** [review]

**Review by bhavin192 — APPROVED** [review]

**Comment by sumitkharche** [comment]
Thanks @sanketsudake and @bhavin192  for review.

**Comment by chetanpdeshmukh** [comment]
Hello @sumitkharche ,
Thanks again for the PR!

Thank you for contributing to BotKube. Could you please fill out [this form](https://docs.google.com/forms/d/e/1FAIpQLSdULT22oZlpHPMXKVEtV4fZ6R8vdGDC2iFvMhoG6IhZkK1SFA/viewform?usp=sf_link), so we can send the well deserved awesome swags :slightly_smiling_face:

Team BotKube

**Pull request merged**